gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -425,21 +425,21 @@ message VehiclePosition {
   }
   optional CongestionLevel congestion_level = 6;
 
-  // The degree of passenger occupancy of the vehicle or carriage. This field is still
+  // The nominal status of passenger occupancy for the vehicle or carriage. This field is still
```

Comment:
I'd like to see more explicit language here that individual producers may not publish all of these values, and therefore consumers must NOT assume that the enumerations are on a linear scale - for example, with `EMPTY` representing the minimum value and `NOT_ACCEPTING_PASSENGERS` or `FULL` representing the max value. Also that producers must not use enumeration values that don't correspond to actual real-world vehicle occupancy states - for example, a producer must NOT internally take a % full numeric value and evenly map it across the enumerations.

It could also recommend (via `should`) that consumers only represent this information to end users based on the enumeration label and not attempt to further interpret the enumeration. For example, a consumer shouldn't map a `STANDING_ROOM_ONLY` value to a roughly half-way full UI graphic bar. Instead, they should show the text "Standing room only" to end users.

Reply:
Please see my latest commit where I added these details.